packages/subgraph/src/liquidity-rewards/updaters.ts

```diff
@@ -240,13 +242,11 @@ export function linkRewardToBids(rewardAllocation:RewardAllocation) : void {
 
     //check to see if the bid is eligible for the reward
     
-    if( //make this a function later
+    if( 
+      allocationStatusToEnum(rewardAllocation.status) == AllocationStatus.Active &&
        bidIsEligibleForReward(bid,rewardAllocation) 
       ){
-
-        appendAllocationRewardToBidParticipants(bid,rewardAllocation);
-
-      
+        appendAllocationRewardToBidParticipants(bid,rewardAllocation);      
```

Comment:
we should also check the allocation strategy. 

if the strategy is for `Lender` then all **accepted** bids are valid

if the strategy is for `Borrower` then only **repaid** bids are valid

Reply:
we may need to attach the reward allocation to bids for the `Borrower` strategy elsewhere like in the handler for the `BidRepaid` event

Reply:
> 

wait why would we need to do this?